### PR TITLE
almanac nouns in almanac.lic

### PR DESCRIPTION
**Pull request by Hiinky**

### CODE REVIEW

**Comment by Hiinky** [comment]
Updated for almanac_noun

File: almanac.lic [hunk 1]
```diff
@@ -12,6 +12,8 @@ class Almanac
     UserVars.almanac_last_use ||= Time.now - 600
     @no_use_scripts = settings.almanac_no_use_scripts
     @almanac_skills = settings.almanac_skills
+    @almanac = settings.almanac_noun
+    @almanac_container = settings.almanac_container
```
Comment (rpherbig) [inline almanac.lic]: ❓ Where is this used?

Comment (Hiinky) [reply almanac.lic]: Ahhhh this this making me crazy.   Finished the rest of the changes.

Comment (Hiinky) [reply almanac.lic]: aborted the almanac_container.  Forgot to remove it here.

**Pull request merged**